data/json/professions.json

```diff
@@ -3417,5 +3417,460 @@
+        "points": 1,
+        "skills": [
+            {
+                "level": 1,
+                "name": "cooking"
+            }
+        ],
+        "items": {
+            "both": [
+                "tshirt",
+                "jacket_light",
+                "jeans",
+                "socks",
+                "sneakers",
+                "knit_scarf",
+                "glasses_safety",
+                "chemistry_set",
+                "textbook_chemistry"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "panties"
+            ]
+        }
+    },
+    {
+        "type": "profession",
+        "ident": "av_club_mem",
+        "name": "A/V Club Member",
+        "description": "You were a member of the school A/V club. You're sure there's some way you can use your technical skills to help stay alive. You just haven't figured out how to make an awesome death ray yet.",
+        "points": 1,
+        "skills": [
+            {
+                "level": 1,
+                "name": "electronics"
+            },
+            {
+                "level": 1,
+                "name": "computer"
+            }
+        ],
+        "items": {
+            "both": [
+                "linuxtshirt",
+                "jacket_light",
+                "jeans",
+                "socks",
+                "sneakers",
+                "knit_scarf",
+                "backpack",
+                "mag_electronics"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "panties"
+            ]
+        }
+    },
+    {
+        "type": "profession",
+        "ident": "janitor",
+        "name": "Janitor",
+        "description": "You earned a living from sweeping up chocolate wrappers and picking chewing gum from under tables. Now the only thing you'll be sweeping are the brains of the dead.",
+        "points": -1,
+        "items": {
+            "both": [
+                "tshirt",
+                "jacket_light",
+                "pants",
+                "socks",
+                "boots",
+                "mop"
+            ],
+            "male": [
+                "boxer_shorts"
+            ],
+            "female": [
+                "bra",
+                "boxer_shorts"
+            ]
+        }
+    },
+    {
+        "type": "profession",
+        "ident": "teacher",
+        "name": "Teacher",
+        "description": "You've been teaching kids for the whole of your life, and they've mostly listened to your teachings. However, the dead won't write out lines for eating you alive.",
+        "points": 0,
+        "skills": [
+            {
+                "level": 3,
+                "name": "speech"
```

Comment:
Speech may not be all that useful, but this would cost 2 chargen points to buy (first chargen point buys rank 2, next buys rank 3).  You're going to need to pay at least 1 for the profession.